src/widgets/connections/ui/connectionList.tsx

```diff
@@ -0,0 +1,61 @@
+import { Card } from "@mui/material";
+import { useEffect, useState } from "react";
+import {
+    ListContextProvider,
+    Pagination,
+    Title,
+    useGetList,
+} from "react-admin";
+import useConnectionsList from "src/hooks/useConnectionsList";
+import useLocalStoreCurrentGroup from "src/hooks/useLocalStoreCurrentGroup";
+import Error from "src/shared/ui/error";
+import Loading from "src/shared/ui/loading";
+import Warning from "src/shared/ui/warning";
+import Connections from "src/widgets/connections/ui/connectionListElement";
+
+const ConnectionList = () => {
+    const [currentGroup] = useLocalStoreCurrentGroup();
+    const [page, setPage] = useState(1);
+    const [perPage, setPerPage] = useState(10);
+    const [, setConnectionList] = useConnectionsList();
+
+    const { data, total, isLoading, error } = useGetList("connections", {
+        meta: { currentGroupId: currentGroup.id },
```

Comment:
```suggestion
        meta: { groupId: currentGroup.id },
```

nit: API не интересует, текущая группа или не текущая. Ему нужна просто группа